spotify/Makefile

```diff
@@ -2,6 +2,10 @@
 run dev:
 	python3 main.py
 
+.PHONY: test
+test:
+	python3 -m pytest
+
```

Comment:
I want to run unit tests with the pipelines now.
To keep things simple I decided to add this test target here, even tough it is unrelated to the rest of the changes